src/acl/SourceAsn.h

```diff
@@ -9,17 +9,22 @@
 #ifndef SQUID_ACL_SOURCEASN_H
 #define SQUID_ACL_SOURCEASN_H
 
-#include "acl/Strategy.h"
-#include "ip/Address.h"
+#include "acl/Data.h"
+#include "acl/ParameterizedNode.h"
+#include "ip/forward.h"
 
-class ACLChecklist;
-
-class ACLSourceASNStrategy : public ACLStrategy<Ip::Address>
+namespace Acl
 {
 
+/// a "src_as" ACL
+class SourceAsnCheck: public ParameterizedNode< ACLData<Ip::Address> >
```

Comment:
This PR has to modify this and similar ACLFooStrategy class names because they are no longer Strategy-derived classes. We are using this renaming as an excuse to also move these classes into the Acl namespace. We should still use a common class naming pattern, of course. This PR uses a Check suffix, so ACLFooStrategy becomes Acl::FooCheck. If you have better naming pattern ideas, please discuss!